atom/browser/api/atom_api_menu_views.cc

```diff
@@ -38,21 +43,32 @@ void MenuViews::PopupAt(Window* window, int x, int y, int positioning_item) {
     location = gfx::Point(origin.x() + x, origin.y() + y);
   }
 
+  int flags = MenuRunner::CONTEXT_MENU |
+              MenuRunner::HAS_MNEMONICS |
+              MenuRunner::ASYNC;
```

Comment:
ASYNC flag is redundant since 59
see https://chromium.googlesource.com/chromium/src.git/+/5ab7c2d09320aa6f4fc6ba0118908ac7e378e77b